projects/novo-elements/src/elements/picker/extras/base-picker-results/BasePickerResults.ts

```diff
@@ -66,7 +66,7 @@ export class BasePickerResults {
   }
 
   set term(value) {
-    if (value !== this._term || this.page === 0) {
+    if (value !== this._term || this.page === 0 || value === '') {
```

Comment:
@amrutha-rajiv this is the fix I found for the issue I showed you. I'm not sure if there is a better approach. The `billingProfile` picker behaves nicely because it retains its `this._term` and so it enters this if block when `value` is an empty string. 
Whereas for location, `this._term` is an empty string *before the user searches a second time*, so it does not enter this if block to process the search. My hunch is that this is a result of the field interaction that clears out `billingLocation` and is somehow resetting the `this._term` to an empty string. 
I'm not sure what else I could compare on here because the property that changes (and should indicate that we need to do a new search) is `[picker].config.options`, and it is simply replaced with a new function that has a different filter.

Reply:
An alternative to this change might be to override `term`'s setter in `location-picker-results` to do some kind of a diff on the options function? LMK what you think.